File: Compiler_Design_Lab/LMS_Virtual_Programming/Ex_1/count.c
/*
Question:
Write a C program to count the number of alphabets in the given string.
*/

#include<stdio.h>
#include<string.h>
int main(){
    int i, n,count=0;
    char st[100];
    scanf("%s",st);
    n=strlen(st);
    for(i=0;i<n;i++){
        if ((st[i]>='a' && st[i]<='z') || (st[i]>='A' && st[i]<='Z')){
            count+=1;
        }
    }
    printf("%d",count);
}

File: ML_&_Data_Analytics_Lab/Cycle_1_Experiments/Ex_4/count.py
'''
Question:
From the string input, count the special characters, alphabets, digits, lowercase and uppercase characters.
Input: Sathyabama 2019 @ 
Output:
Digits: 			4
Alphabets: 		   10
Special Characters: 1
Lowercase: 		    9
Uppercase: 		    1
'''

def Count(str):
    upper, lower, number, special = 0, 0, 0, 0
    for i in range(len(str)):
        if (ord(str[i]) >= 65 and ord(str[i]) <= 90): 
            upper += 1 
        elif (ord(str[i]) >= 97 and ord(str[i]) <= 122): 
            lower += 1 
        elif (ord(str[i]) >= 48 and ord(str[i]) <= 57): 
            number += 1 
        else: 
            special += 1
        alphabets = upper+lower

    print('Digits:', number)
    print('Alphabets:', alphabets)
    print('Special characters:', special)
    print('Lower case:', lower)
    print('Upper case:', upper) 
     
    
     

str = input("Enter the String:") 
Count(str) 
